sunpy/net/hek/util.py

```diff
@@ -0,0 +1,150 @@
+import os
```

Comment:
It should it be util or utils? How are the other files like this named around sunpy?

Reply:
Yeah I think it should be `utils.py`, like `sunpy.coordinates.utils`, `sunpy.coordinates.wcs_utils` and `sunpy.map.maputils.py`.

Reply:
Thanks.